detect_secrets_server/storage/base.py

```diff
@@ -7,11 +7,13 @@
 from detect_secrets.core.log import log
 
 from .core import git
+from detect_secrets_server.util.version import is_python_2
 
-try:
-    FileNotFoundError
-except NameError:
+if is_python_2():   # pragma: no cover
```

Comment:
nit: using a `try...except` is the "Pythonic" way to do this

Reply:
It is, but I ended up using a similar logic in a variety of places (i.e. tests \[ [1](https://github.com/Yelp/detect-secrets-server/pull/14/files#diff-50674fdf7faecdcbc77a95e6c3fce035R19) ] \[ [2](https://github.com/Yelp/detect-secrets-server/pull/14/files#diff-32c5368e62a45ec685dedc5679f0bbb6R15) ]), that I figure it's better to DRY it up.